tests/phpunit/includes/wp-screen-mock.php

```diff
@@ -10,6 +10,34 @@
  * Inspired by @see https://gist.github.com/DragonBe/24761f350984c35b73966809dd439135
  */
 class Wp_Screen_Mock {
```

Comment:
Can we rename it to `WP_Screen_Mock` and do the same where it's used? I noticed it's sometimes `WP_Screen_Mock` and sometimes `Wp_Screen_Mock`.

Reply:
Actually I did the other way: I kept `Wp_Screen_Mock` (to differ from WordPress' `WP_*` classes) and renamed it in several places in PLL Pro.
Do you prefer `WP_Screen_Mock` over `Wp_Screen_Mock`? On second thought it makes sense to use `WP_Screen_Mock` indeed, like in `WP_Screen` + `_Mock`.

Reply:
Renamed in [7ab744a](https://github.com/polylang/polylang/pull/1143/commits/7ab744a61bf13f929b7c62a128ceaaf8a5404e6c).